protocol/blocks.go

```diff
@@ -0,0 +1,42 @@
+package protocol
+
+import (
+	peerpkg "github.com/kaspanet/kaspad/protocol/peer"
+	"github.com/kaspanet/kaspad/util"
+	"github.com/kaspanet/kaspad/util/daghash"
+	"github.com/kaspanet/kaspad/wire"
+	"sync/atomic"
+)
+
+// OnNewBlock updates the mempool after a new block arrival, and
+// relays newly unorphaned transactions and possibly rebroadcast
+// manually added transactions when not in IBD.
+// TODO(libp2p) Call this function from IBD as well.
+func (m *Manager) OnNewBlock(block *util.Block) error {
+	acceptedTxs, err := m.txPool.HandleNewBlock(block)
```

Comment:
Rename somehow that it's clear these are transactions that were accepted to the mempool.